### DISCUSSION TITLE
Geforce driver 7.76 faster than anything newer?

### DISCUSSION THREAD

**Discussion by Skalabala** [86Box/86Box#4443] [Q&A] [answered]
Hello everyone :) This might be an oddball question...
So I have a hobby to play with old computers, specifically Super Socket 7 with ALi chipsets.
With my years of testing Geforce driver 7.76 to12.08 is the fastest but it only has support up to Geforce 3.
There is a file in the driver named ALiPCIMP.pci and any newer driver that does not have this is slower by up to 30%
I would like to have a fast driver that supports a Geforce 4 Ti.
Now how do I figure out how to "imbed" this ALiPCIMP.pci into a newer driver?

**Comment by jriwanek** [comment, marked as answer]
As there are no nvidia or Geforce cards in 86box, you've clearly come to the wrong place.
